test/storage/checkpoint_test.cpp

```diff
@@ -0,0 +1,194 @@
+#include <algorithm>
+#include <iostream>
+#include <random>
+#include <string>
+#include <vector>
+#include "common/object_pool.h"
+#include "storage/checkpoint_manager.h"
+#include "storage/sql_table.h"
+#include "storage/storage_util.h"
+#include "util/random_test_util.h"
+#include "util/storage_test_util.h"
+#include "util/transaction_test_util.h"
+
+#define CHECKPOINT_FILE_PREFIX "checkpoint_"
+
+namespace terrier {
+
+struct CheckpointTests : public TerrierTest {};
+
+// NOLINTNEXTLINE
+TEST_F(CheckpointTests, SimpleCheckpointNoVarlen) {
```

Comment:
This test (and the next) are not actually testing anything other than that your code doesn't segfault.   To unit test checkpoint you should be verifying that your serialized format looks like what your commented spec says it should.

Reply:
You are right. These 2 tests were written when there was no recovery. Already removed.